src/pages/Print/components/DownloadDialog.tsx

```
@@ -0,0 +1,285 @@
+  const blob = await new Promise((resolve: BlobCallback) => {
+    canvas.toBlob(resolve, 'image/png')
+  });
+
+  document.body.removeChild(img);
+  document.body.removeChild(copiedNode);
+  return blob;
+}
+
+async function downloadCards(fileName: string, settings: PrintSettingsType) {
+  if (!fileName || !settings) return;
+  const { doubleCard, withBleed, width, height } = settings;
```

ESLint

[error] 'doubleCard' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)